doc/modules/cross_validation.rst

```diff
@@ -119,8 +118,8 @@ method of the estimator. It is possible to change this by using the
 scoring parameter::
 
   >>> from sklearn import metrics
-  >>> scores = cross_validation.cross_val_score(clf, iris.data, iris.target,
-  ...     cv=5, scoring='f1_weighted')
+  >>> scores = cross_val_score(
+  ...     clf, iris.data, iris.target, cv=5, scoring='f1_weighted')
   >>> scores                                              # doctest: +ELLIPSIS
   array([ 0.96...,  1.  ...,  0.96...,  0.96...,  1.        ])
```

Comment:
Why is this doctest skipped? Isn't it determnistic?

It looks weird that the values are the same for the default score (accuracy) and this one (`f1_weighted`).